contracts/TrueCurrencies/TrueRewardBackedToken.sol

```diff
@@ -400,4 +341,12 @@ contract TrueRewardBackedToken is RewardTokenWithReserve {
         delete _rewardDistribution[msg.sender][0];
         _rewardDistribution[msg.sender].length--;
     }
+
+    function neitherSideHasTrueRewards(address from, address to) internal view returns (bool) {
```

Comment:
eliminate this function

Reply:
You really hate extracting functions, don't you :)